server/src/main/java/org/apache/iotdb/db/qp/physical/sys/SetTTLPlan.java

```diff
@@ -54,6 +54,11 @@ public SetTTLPlan(String storageGroup) {
     return null;
   }
 
+  @Override
+  public List<String> getPathsStrings() {
+    return null;
```

Comment:
And here.

Reply:
I'll change it. Thanks for your suggestion.